README.md

```diff
@@ -168,6 +168,15 @@ and `END` texts and line breaks (`\n`). Below an example `.env` file:
 ED25519KEY="-----BEGIN PRIVATE KEY-----\nMC4CAQAwBQYDK2VwBCIEIB8nP5PpWU7HiILHSfh5PYzb5GAcIfHZ+bw6tcd/LZXh\n-----END PRIVATE KEY-----"
 ```
 
+### `integrityBlockSign.isIwa`
+
+Type: `boolean`
+
+If `undefined` or `true`, enforces certain IWA-related checks for the headers.
+Also if `headerOverride` together with the previous is undefined, adds default
+IWA headers. If set to `false`, skips validation checks and doesn't tamper with
+the headers.
```

Comment:
Oh, I thought you were going to also inject the IWA headers even when `headerOverride` is used. But I guess this way works as well.

Reply:
I can add that still!